packages/static-wado-webserver/lib/stowrs.mjs

```diff
@@ -0,0 +1,60 @@
+import formidable from "formidable";
+import childProcess from "child_process";
+import util from "util";
+import fs from "fs";
+
+const exec = util.promisify(childProcess.exec);
+
+/**
+ * Handles an incoming stow-rs POST data, either in application/dicom (single instance), or in
+ * multipart/related with JSON data elements.
+ *
+ * TODO: Handle bulkdata and images, in addition to the raw JSON data.
+ */
+
+const stowrsGenerator = (params) => {
+  const { stowCommands, verbose } = params;
+  const createCommandLine = (files, commandName) => files.reduce((p, c) => `${p} ${c.filepath}`, commandName);
+
+  const stowrs = (req, res, next) => {
+    const form = formidable();
+    form.parse(req, (err, fields, files) => {
```

Comment:
You are assuming files when parsing, so I would say we should configure formidable as:

`formidable({multiples: true})`